src/main/groovy/com/rundeck/plugins/ansible/ansible/AnsibleVault.java

```diff
@@ -62,6 +70,11 @@ public String encryptVariable(String key,
         if (ansibleBinariesDirectory != null) {
             ansibleCommand = Paths.get(ansibleBinariesDirectory.toFile().getAbsolutePath(), ansibleCommand).toFile().getAbsolutePath();
         }
+
+        if(processExecutorBuilder==null){
+            processExecutorBuilder = ProcessExecutor.builder();
+        }
+
         procArgs.add(ansibleCommand);
```

Comment:
nit - This could be:
```
List<String> procArgs = List.of(ansibleCommand, "encrypt_string", "--vault-id", "internal-encrypt@" + vaultPasswordScriptFile.getAbsolutePath())
```